src/EditorFeatures/Core/ChangeSignature/AbstractChangeSignatureCommandHandler.cs

```diff
@@ -131,7 +131,7 @@ private static void HandleResult(ChangeSignatureResult result, Solution oldSolut
                 // wait context. That means the command system won't attempt to show its own wait dialog 
                 // and also will take it into consideration when measuring command handling duration.
                 context.OperationContext.TakeOwnership();
-                finalSolution = previewService.PreviewChanges(
```

Comment:
This is the only synchronous application, and it's on a command handler. I think we should move this to async...

Reply:
They would be nice:-)

Reply:
Can this code just do a jtf run?